README.md

```diff
@@ -1,15 +1,23 @@
 # Backup and Restore Operator
 
-### Latest release
+## Latest release
 
 [![Latest](https://img.shields.io/badge/dynamic/yaml?label=backup-restore-operator&query=%24.entries%5B%27rancher-backup%27%5D%5B0%5D.appVersion&url=https%3A%2F%2Fcharts.rancher.io%2Findex.yaml)](https://github.com/rancher/backup-restore-operator/releases/latest)
 
-### Description
+## Description
 
-* This operator provides ability to backup and restore Kubernetes applications (metadata) running on any cluster. It accepts a list of resources that need to be backed up for a particular application. It then gathers these resources by querying the Kubernetes API server, packages all the resources to create a tarball file and pushes it to the configured backup storage location. Since it gathers resources by querying the API server, it can back up applications from any type of Kubernetes cluster.
-* The operator preserves the ownerReferences on all resources, hence maintaining dependencies between objects.
-* It also provides encryption support, to encrypt user specified resources before saving them in the backup file. It uses the same encryption configuration that is used to enable [Kubernetes Encryption at Rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/). Follow the steps in [this section](https://ranchermanager.docs.rancher.com/reference-guides/backup-restore-configuration/backup-configuration#encryption) to configure this.
+The Backup and Restore Operator provides ability to back up and restore the Rancher application running on any Kubernetes cluster.
 
+### Use-Cases
+- Backing up Rancher before Rancher upgrades and restoring after a failed one.
+- Restoring your *Rancher application* to a new cluster in a DR scenario.
```

Comment:
I don't think you need to highlight this

Reply:
And "DR" should prob be spelled out as it's not defined earlier

Reply:
Will opt to expand DR and include this item - I know it's a topic that comes up in support world. So mainly defining that here for clarity on that.

Reply:
Right, we need to emphasize that it's the Rancher application getting backed up, and not user data